example/35-namespacedcloudprofile.yaml

```
@@ -9,25 +9,43 @@ spec:
   parent:
     kind: CloudProfile
     name: local # could be any CloudProfile
-# kubernetes: # overwrite kubernetes versions if necessary
-  machineImages:
-  - name: suse-chost-1
-    versions:
-    - version: 15.4.20220818
-    - version: 15.4.20220620
-      expirationDate: "2023-02-28T23:59:59Z"
-  - name: ubuntu-custom
+  kubernetes:
     versions:
-    - version: 18.04.201906170
+    - version: 1.29.0
+      expirationDate: "2025-02-28T23:59:59Z"
   machineTypes:
-  - name: m5.xlarge
+  - name: local-large
     cpu: "4"
     gpu: "0"
     memory: 16Gi
     usable: true
-# volumeTypes: # add additional volume types if necessary
-# CA bundle that will be installed onto every shoot machine that is using this provider profile.
-# caBundle: |
-#   -----BEGIN CERTIFICATE-----
-#   ...
-#   -----END CERTIFICATE-----
+  machineImages:
+  - name: local
+    versions:
+    - version: 1.0.1-dev
+      architectures:
+      - amd64
+      cri:
+      - name: containerd
+    - version: 1.0.0
+      expirationDate: "2024-12-31T23:59:59Z"
+  - name: ubuntu-custom
+    versions:
+    - version: 18.04.201906170
+      architectures:
+      - amd64
+      cri:
+      - name: containerd
+  providerConfig: # add provider specific configuration if necessary, especially for custom machine images
```

```suggestion
  providerConfig:
```

Probably clear on its own.